apps/advisor/pkg/app/checks/plugincheck/check.go

```diff
@@ -44,12 +48,38 @@ func (c *check) Name() string {
 	return "plugin"
 }
 
+type pluginItem struct {
+	Plugin *pluginstore.Plugin
+	Err    *plugins.Error
+}
+
 func (c *check) Items(ctx context.Context) ([]any, error) {
 	ps := c.PluginStore.Plugins(ctx)
-	res := make([]any, len(ps))
-	for i, p := range ps {
-		res[i] = p
+	resMap := map[string]*pluginItem{}
+	for _, p := range ps {
+		resMap[p.ID] = &pluginItem{
+			Plugin: &p,
+			Err:    c.pluginErrorResolver.PluginError(ctx, p.ID),
+		}
+	}
+
+	pluginErrors := c.pluginErrorResolver.PluginErrors(ctx)
```

Comment:
I assume this is adding results for plugins that don't exist in the store? Might be good to add a comment in that case!